requirements.txt

```diff
@@ -1,6 +1,33 @@
-matplotlib==3.0.2
-PySide==1.2.4
-# PySide2==5.12.0
-Shiboken==1.2.2
-six==1.12.0
-Markdown==3.2.2
+area==1.1.1
+bzr_pqm==1.4.0
+cog==0.6.1
+ConfigParser==5.3.0
+defusedxml==0.7.1
+GitPython==3.1.31
+ifcopenshell==0.7.0.230318
+kdtree==0.16
+ladybug==0.0.2
+matplotlib==3.6.3
+numpy==1.24.2
+ocl==0.12
+opencamlib==2023.1.11
+packaging==23.0
+Pivy==0.6.8
+ply==3.11
+ptvsd==4.3.2
+pyNastran==1.3.4
+PyQt4==4.11.4
+PyQt5==5.15.9
+pyshp==2.3.1
+PySide2==5.15.2.1
+Pysolar==0.10
+pysolar==0.10
+PyYAML==6.0
+requests==2.28.2
+rpdb2==2.0.0.1.2
+sets==0.3.2
+six==1.16.0
+vermin==1.5.1
+lazy_loader==0.1
+BOPTools==0.12
```

Comment:
this one looks like it is a module from FreeCAD itself so I don't know if it makes sense putting it here
https://github.com/FreeCAD/FreeCAD/tree/master/src/Mod/Part/BOPTools